lib/pubsub/subscription.js

```diff
@@ -267,12 +270,13 @@ function Subscription(pubsub, options) {
 
   this.autoAck = is.boolean(options.autoAck) ? options.autoAck : false;
   this.closed = true;
-  this.interval = is.number(options.interval) ? options.interval : 10;
   this.inProgressAckIds = {};
+  this.interval = is.number(options.interval) ? options.interval : 10;
   this.maxInProgress =
     is.number(options.maxInProgress) ? options.maxInProgress : Infinity;
   this.messageListeners = 0;
   this.paused = false;
+  this.timeout = options.timeout;
```

Comment:
Do we need to provide a default? Or if it's null/undefined a timeout won't occur?